### ISSUE TITLE
Odd Debian VM Issues

### ISSUE THREAD

**Issue by e6lk7dqzm83p** [QubesOS/qubes-issues#4159]
### Qubes OS version: R4.0
<!-- (e.g., `R3.2`)

### Affected Components
Debian VMs

###Steps to reproduce behavior
Unknown, these issues seem to have been present since I installed Qubes

###Expected Behavior
Debian App and Template VMs behave in the same way as Fedora App and Template VMs

###Actual Behavior

-I cannot make my debian AppVM into a disposable VM; I have tried uninstalling and reinstalling the template

- Debian AppVMs don't seem as stable as my Fedora ones, it's hard for me to quantify this

-When I exit the debian-9 TemplateVM it removes all my USB devices from sys-usb (though at least some of the devices, e.g. my keyboard and mouse, still work)

-Video seems to be poorer in Debian vs Fedora

-Sometimes commands I put in the dom0 terminal don't return and bring back to a probmp (e.g., launching the terminal in non-Fedora based AppVMs)

-The AppVM manager in the system tray (the Q icon) doesn't seem to be able to launch the terminal in Debian based AppVMs

**Comment by andrewdavidwong** [comment]
Based on our [issue reporting guidelines](https://www.qubes-os.org/doc/reporting-bugs/), this does not appear to be suitable for `qubes-issues`. We ask that you please send this to the `qubes-users` [mailing list](https://www.qubes-os.org/support/) instead. If, after reading our [issue reporting guidelines](https://www.qubes-os.org/doc/reporting-bugs/), you believe we are mistaken, please leave a brief comment explaining why. We'll be happy to take another look, and, if appropriate, reopen this issue. Thank you for your understanding.

**Comment by e6lk7dqzm83p** [comment]
This appears to be a specific issue related to Debian VMs which are not behaving as expected.  There are many symptoms of this, as I posted, including the fact that I cannot create DispVMs.  While it is possible each of these issues is unrelated the fact that they seem to occur only with Debian seems to indicate a broader issue.

I have read the documentation around these issues in the roughly two weeks I spent setting up my system and am now posting this based on feedback I've gotten from other Qubes users: there appears to be a bug.
